### PR TITLE
Tweak CSS for Create a List modal, Add id for new posted list, Fix re…

### PR DESCRIPTION
**Pull request by paprikatie**
…sizing of overall image carousel

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -10,16 +11,20 @@ import CreateModal from './CreateModal.jsx';
...
 const Body = styled.div`
-  background-color: rgb(255, 255, 255);
+  // background-color: rgb(255, 255, 255);
   display: flex;
   flex-direction: column;
   overflow-y: scroll;
   max-width: 1110px;
   margin: auto;
   justify-content: center;
+  align-items: stretch;
```
Comment (chrsbell) [inline client/src/components/App.jsx:27]: this is smart

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -182,7 +188,7 @@ class App extends React.Component {
     const { modal } = this.state;
     this.setState({
       modal: !modal,
-      imageUrl: imageUrl,
+      imageUrl,
```
Comment (chrsbell) [inline client/src/components/App.jsx:191]: just started doing this too

File: client/src/components/ListEntry.jsx [hunk 3]
```diff
@@ -23,6 +24,8 @@ const ImgFlexRow = styled.div`
   height: 180px;
   position: sticky;
   margin-bottom: 9px;
+  scroll-snap-type: x mandatory;
```
Comment (chrsbell) [inline client/src/components/ListEntry.jsx:27]: lets go

File: client/src/components/App.jsx (hunk 1 above)
Comment (paprikatie) [reply client/src/components/App.jsx:27]: thanks! took it from Airbnb :)

File: client/src/components/App.jsx (hunk 2 above)
Comment (paprikatie) [reply client/src/components/App.jsx:191]: Airbnb style guide keeps me grounded

File: client/src/components/ListEntry.jsx (hunk 3 above)
Comment (paprikatie) [reply client/src/components/ListEntry.jsx:27]: ^^ it doesnt work ^^;

**Review by chrsbell — APPROVED** [review]
Looks good

**Pull request merged**